### PR TITLE
feat(dlp): Split jobs samples into multiple files such that each region tag gets its own file.

### PR DESCRIPTION
**Pull request by Avani-Thakker-Crest**
## Description
Fix PR as part of the issue: https://github.com/GoogleCloudPlatform/python-docs-samples/issues/10601
This PR only splits jobs samples into multiple files.
Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: dlp/snippets/Jobs/create_job.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+"""Sample app to list and delete DLP jobs using the Data Loss Prevent API. """
+
+from __future__ import annotations
+
+import argparse
+
+import google.cloud.dlp
+
+
+# [START dlp_create_job]
```
Comment (rsamborski) [inline dlp/snippets/Jobs/create_job.py:24]: nit: please make sure required imports are inside the region_tag (`google.cloud.dlp` in this case)

File: dlp/snippets/Jobs/delete_job.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+"""Sample app to list and delete DLP jobs using the Data Loss Prevent API. """
+
+from __future__ import annotations
+
+import argparse
+
+import google.cloud.dlp
+
+
+# [START dlp_delete_job]
```
Comment (rsamborski) [inline dlp/snippets/Jobs/delete_job.py:24]: nit: please make sure required imports are inside the region_tag (`google.cloud.dlp` in this case)

File: dlp/snippets/Jobs/get_job.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+"""Sample app to list and delete DLP jobs using the Data Loss Prevent API. """
+
+from __future__ import annotations
+
+import argparse
+
+import google.cloud.dlp
+
+
+# [START dlp_get_job]
```
Comment (rsamborski) [inline dlp/snippets/Jobs/get_job.py:24]: nit: please make sure required imports are inside the region_tag (`google.cloud.dlp` in this case)

File: dlp/snippets/Jobs/list_jobs.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+from __future__ import annotations
+
+import argparse
+
+from typing import Optional
+
+import google.cloud.dlp
+
+
+# [START dlp_list_jobs]
```
Comment (rsamborski) [inline dlp/snippets/Jobs/list_jobs.py:26]: nit: please make sure required imports are inside the region_tag (`google.cloud.dlp` in this case)

**Review by rsamborski — APPROVED** [review]
Approved with couple nits.

File: dlp/snippets/Jobs/create_job.py (hunk 1 above)
Comment (Avani-Thakker) [reply dlp/snippets/Jobs/create_job.py:22]: done

File: dlp/snippets/Jobs/delete_job.py (hunk 2 above)
Comment (Avani-Thakker) [reply dlp/snippets/Jobs/delete_job.py:22]: done

File: dlp/snippets/Jobs/get_job.py (hunk 3 above)
Comment (Avani-Thakker) [reply dlp/snippets/Jobs/get_job.py:22]: done

**Review by catherinerosesantos — APPROVED** [review]
Thanks for letting me know. The docs will need to be updated. Approving this now so the doc updates can be tested and published.

**Pull request merged**